server/routes/sessions.js

```diff
@@ -44,20 +44,86 @@ const sessions = [
         sport: "Soccer",
         joined: false,
         dateTime:" dayjs('2022-04-17T15:30')",
-        type: "outdoor,"
+        type: "outdoor"
     },
+    {
+      name: "Basketball",
+      description: "play time",
+      city: "Vancouver",
+      location: "MacInnes Field",
+      equipment: "Frisbee",
+      playersNeeded: 3,
+      groupId: 3,
+      image: image,
+      sport: "Basketball",
+      joined: false,
+      dateTime: "dayjs('2022-04-17T15:30')"
```

Comment:
I didn't put an extra effort to correct the dummy data. It is not necessary because all of this data will be removed when we will set up the database.